### PR TITLE
Feature/54625

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: src/atoms/table/Table.selectors.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": ".table--odd-even",
```
Comment (robwozniak) [inline src/atoms/table/Table.selectors.json]: We don't have any styles for it, are you missing commit something or we're waiting for updated component?

File: src/atoms/table/Table.stories.js [hunk 2]
```diff
@@ -25,7 +21,9 @@ const defaultData = {
 }
 
 storiesOf('Atoms/Table', module)
-  .addParameters({ info })
+  .addParameters({
+    info: generateVueInfoTable(selectorsConfig, 'BEM modifiers')
+  })
   .add('Default', () => ({
     ...defaultData,
```
Comment (robwozniak) [inline src/atoms/table/Table.stories.js]: `Table.selectors.json` contain one class but I can't see props with `classKnobs` regarding possible BEM class options.

File: src/molecules/message/Message.stories.js [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
-  #### Use these modifiers with \`.message\` class.
-  - \`.message--success\` - Selector for applying styles for success message
-  - \`.message--error\` - Selector for applying styles for error message
-  ---
+  ${generateVueInfoTable(selectorsConfig, 'BEM modifiers')}
 `
 
-// @vue/component
 const defaultData = {
   components: { AMessage },
```
Comment (robwozniak) [inline src/molecules/message/Message.stories.js]: We need to add props `classKnobs` and pass options from `Message.selectors.json` to make it visible in storybook app, now they are not available to choose :cry:

**Comment by szafran89** [comment]
@robwozniak for heading and price we have different tasks created.

File: src/atoms/table/Table.selectors.json (hunk 1 above)
Comment (szafran89) [reply src/atoms/table/Table.selectors.json]: Related styles are under `TableRow.scss`

File: src/atoms/table/Table.stories.js (hunk 2 above)
Comment (szafran89) [reply src/atoms/table/Table.stories.js]: added

File: src/molecules/message/Message.stories.js (hunk 3 above)
Comment (szafran89) [reply src/molecules/message/Message.stories.js]: done

**Pull request merged**